packit.yaml

```diff
@@ -34,6 +35,21 @@ jobs:
       - centos-stream-9-x86_64
       - centos-stream-9-aarch64
 
+  # current Fedora runs reverse dependency testing against https://copr.fedorainfracloud.org/coprs/g/cockpit/main-builds/
+  - job: tests
+    identifier: revdeps
+    trigger: pull_request
+    targets:
+      - fedora-stable
```

Comment:
This triggers F37?! That sounds wrong?

Reply:
Ah it does, both F37 and F38.

Reply:
Eww yes, apparently that triggers 37 *and* 38. I [previously had](https://github.com/cockpit-project/cockpit/compare/8a6c4757ccb6b5a5c9037249253244273cde93d9..1002ce741ab673726068213a38022e60a58ddbb4) "fedora-38" hardcoded, but that's ugly. I think there's something like "fedora-latest" or so, let me check.